src/main/java/pl/company/foodatu/meals/domain/MealsConfiguration.java

```diff
@@ -3,18 +3,19 @@
 import org.springframework.context.annotation.Bean;
 import org.springframework.context.annotation.Configuration;
 import org.springframework.context.annotation.Profile;
+import pl.company.foodatu.meals.infrastructure.MealPublisher;
 
 @Configuration
 class MealsConfiguration {
     @Bean
     @Profile("local")
-    MealsFacade mealsInMemoryFacade() {
-        return new MealsFacade(new InMemoryMealsRepository(), new InMemoryStdProductsRepository());
+    MealsFacade mealsInMemoryFacade(MealPublisher mealPublisher) {
```

Comment:
So on the `local` profile you want to use actual kafka implementation? That's fine, but we could also use inmemory one

Reply:
I wanted to have access to the InMemoryMealPublisher in the unit tests, so I can poll events from the queue and assert them. But I didn't think about local profile, so I should reconsider this configuration class.